com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/HDSubTarget.cs

```diff
@@ -119,6 +121,9 @@ public override void Setup(ref TargetSetupContext context)
 
         protected SubShaderDescriptor PostProcessSubShader(SubShaderDescriptor subShaderDescriptor)
         {
+            if (VFXSubTarget.IsConfigured())
+                subShaderDescriptor = VFXSubTarget.PostProcessSubShader(subShaderDescriptor);
```

Comment:
Here is the main entry point in HDRP to invoke VFX to post process the current sub target.